Mango API/src/com/infiniteautomation/mango/rest/latest/websocket/MangoWebSocketHandshakeHandler.java

```diff
@@ -21,24 +21,32 @@
  */
 @Component("mangoWebSocketHandshakeHandlerV2")
 public class MangoWebSocketHandshakeHandler extends DefaultHandshakeHandler {
+    private static final int KB = 1024;
+    private static final long SECOND = 1000L;
 
-    //TODO Autowire properties
-    public static final int DEFAULT_INPUT_BUFFER_SIZE = 8192;
-    public static final long DEFAULT_IDLE_TIMEOUT_MS = 60000L;
-    
     public MangoWebSocketHandshakeHandler(@Autowired ServletContext servletContext) {
         super(new JettyRequestUpgradeStrategy(new WebSocketServerFactory(servletContext, getPolicy())));
     }
-    
+
     static WebSocketPolicy getPolicy() {
-        int inputBufferSize = Common.envProps.getInt("web.websocket.inputBufferSize", DEFAULT_INPUT_BUFFER_SIZE);
-        long idleTimeout = Common.envProps.getLong("web.websocket.idleTimeoutMs", DEFAULT_IDLE_TIMEOUT_MS);
+        int inputBufferSize = Common.envProps.getInt("web.websocket.inputBufferSize", 8 * KB);
```

Comment:
Inject `org.springframework.core.env.Environment` into the constructor and get properties from `Environment`